qiskit/pulse/utils.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
```

Comment:
Why was this moved? Perhaps it was left behind when the scheduler move was reverted?

Reply:
`format_meas_map` was only required for the measurement macros. Since these now live in pulse this utility should as well.